latte/models/probabilistic_pca/probabilistic_pca.py

```diff
@@ -64,6 +71,8 @@ def generate(
     Returns:
         Populated GaussianGenerativeDataset
     """
+    if not d_measured:
```

Comment:
I don't think it'll be troublesome, but in principle checking for None is accomplished by:
`if d_measured is None:`

However, one can use `d_measured=0` and this `if` will be executed (which makes sense). I'd suggest adding an `assert` if you decided that you didn't like this behaviour (or if you want to get rid of negative values)